packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -615,18 +618,94 @@ export const RemoteServerFactory = {
         }
     },
 
-    addSSRRenderer(app) {
-        // See - https://www.npmjs.com/package/webpack-hot-server-middleware#usage
-        const {buildDir} = app.options
-        const _require = eval('require')
-        const serverRenderer = _require(path.join(buildDir, 'server-renderer.js')).default
-        const stats = _require(path.join(buildDir, 'loadable-stats.json'))
+    addStaticAssetServing() {
+        // Handled by the CDN on remote
+    },
+
+    addDevServerGc() {
```

Comment:
```suggestion
    addDevServerGarbageCollection() {
```
`Gc` isn't very descriptive.